File: Python/uberBlack.py
from car import Car

class UberBlack(Car):
    typeCarAccepted = []
    seatsMaterial = []

    def __init__(self, driver, license, passenger, typeCarAccepted, seatsMaterial):
        super().__init__(driver, license, passenger)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial

File: Java/UberBlack.java
import java.util.ArrayList;
import java.util.Map;

public class UberBlack extends Car {
    Map<String, ArrayList<Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;

    public UberBlack(Account driver, String license, 
    Map<String, ArrayList<Integer>> typeCarAccepted,
    ArrayList<String> seatsMaterial){
        super(driver, license);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }
}
